src/shared/analytics/shared/transactionContextToAnalytics.ts

```diff
@@ -106,7 +106,10 @@ export function addressActionToAnalytics({
     asset_name_received: toMaybeArr(incoming?.map(getAssetName)),
     asset_address_sent: toMaybeArr(outgoing?.map(getAssetAddress)),
     asset_address_received: toMaybeArr(incoming?.map(getAssetAddress)),
+    warning_was_shown: warningWasShown,
```

Comment:
Do you send these fields for Approve transactions?
If I remember right, we still need send them, but with the default values